utils/auth.js

```diff
@@ -88,13 +88,13 @@ export default class Auth {
     }
   }
 
-  handleAuthentication(location) {
-    if (typeof localStorage === 'undefined') {
-      return
-    }
+  handleAuthentication() {
     return new Promise((resolve, reject) => {
+      if (typeof localStorage === 'undefined') {
```

Comment:
todo: based on conversations with Jared we should be using cookies to access the auth from the SSR